patches/chrome-browser-resources-settings-appearance_page-appearance_page.html.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.html b/chrome/browser/resources/settings/appearance_page/appearance_page.html
+index 720d25d126a118315a08cef90d77d1fc78775254..5072dfc50d1108b2eea75c6079ae144395d5c04b 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_page.html
++++ b/chrome/browser/resources/settings/appearance_page/appearance_page.html
+@@ -175,6 +175,20 @@
+             </template>
+           </select>
+         </div>
++<if expr="not _google_chrome">
```

Comment:
same with this. Maybe it makes more sense to keep this separate from chrome webstore theme prefs? If we want to keep it under the same heading, seems like we should be able to use the templating engine to insert an html snippet here without patching the whole thing in?

Reply:
`settings-brave-apperance-page` module is only added.